src/widgets/chat-message.jsx

```diff
@@ -1,6 +1,6 @@
 // Single message, sent or received.
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```

Comment:
It does not seem to be needed. The `injectIntl` is needed only when `this.props.intl.formatMessage` is used.